src/providers/hana/qgshanasettings.h

```diff
@@ -32,6 +32,12 @@ struct QgsHanaIdentifierType
   static Value fromInt( uint );
 };
 
+enum class QgsHanaConnectionType : uint
+{
+  HOST_PORT = 0,
```

Comment:
```suggestion
  HostPort = 0,
```

matches standard QGIS code formatting